lib/_cookieUtils.ts

```diff
@@ -49,6 +50,12 @@ export function setUserToken(userToken: string | number): void {
   }
 }
 
-export function getUserToken() {
+export function getUserToken(
+  options?: any,
```

Comment:
Is there a reason why we expose an `options` argument since we don't use it? To make it easier to extend afterward?

Reply:
The only reason is: avoid a breaking change.

When the callback was added on `processQueue` level, it assumed all method had the same signature.
`aa('method', {  /* params */ }, function callback(){...})` .
So when you called `getUserToken`, you had to include and uneeded `params` argument
```js
aa('getUserToken', /* options */ null, (err, userToken) => {...})
```

Now that I moved the callback on the method level (which is the way I should have done it in the first place), we could technically get rid of `options` but that would be a breaking change.